wntr/gis/network.py

```diff
@@ -192,7 +192,7 @@ def _create_wn(self, append=None):
             if element.shape[0] > 0:
                 assert (element['geometry'].geom_type).isin(['Point']).all()
                 df = element.reset_index()
-                df.rename(columns={'index':'name', 'geometry':'coordinates'}, inplace=True)
+                df.rename(columns={index_col:'name', 'geometry':'coordinates'}, inplace=True)
```

Comment:
Here, rather than resetting the index and then renaming the new index column I think it is more straightforward to run `df = element.reset_index(names="name")` which will reset the index and rename the index column to `name` in one line.